### ISSUE TITLE
A question about Loss phenomenon

### ISSUE THREAD

**Issue by A-New-Page** [microsoft/unilm#563]
**Describe**
Model I am using (WavLM):
Sorry to interrupt again! I would like to pre-train a new WavLM model, and i have completed the training process according to normal Pytorch training mode. I also refer to the code of HuBERT. But for the result, my loss is almost no change at all. Have you encountered this phenomenon? Could you please explain it?Thank you!I am new in wav2vec field. Looking forward to you reply!

**Comment by MarkWuNLP** [comment]
Hi,
    Thanks for your interest in WavLM. Could you further clarify your question? What you have done is model pre-training and the loss does not decrease for both HuBERT and WavLM. Is my understanding correct?

**Comment by A-New-Page** [comment]
Yes!Maybe there is a problem with my model or loss function.So, do you have any suggestion?Thanks.

**Comment by MarkWuNLP** [comment]
You can decode the HuBERT released model to check whether your model structure has any issue. If not, you can check the pseudo-labeling file. A correct label file contains labels like ``3 3 3 5 5 5 5 6 7  1 1", where successive tokens appear in an utterance.  If the pseudo-labeling is correct, I suggest you check your training config. You should strictly follow what HuBERT suggests.

**Comment by A-New-Page** [comment]
Thank you! I will try it followed your suggestions! Thanks a lot.
